src/xenia/app/emulator_window.cc

```diff
@@ -186,9 +186,9 @@ bool EmulatorWindow::Initialize() {
     file_menu->AddChild(
         MenuItem::Create(MenuItem::Type::kString, "&Open...", "Ctrl+O",
                          std::bind(&EmulatorWindow::FileOpen, this)));
-    file_menu->AddChild(
-        MenuItem::Create(MenuItem::Type::kString, "Close",
-                         std::bind(&EmulatorWindow::FileClose, this)));
+//    file_menu->AddChild(
```

Comment:
I'd recommend `#if 0` to avoid breaking formatting. I'm not sure if this will even pass clang-format.

Reply:
Like this?
```
      #if 0
      case ui::VirtualKey::kF7: {
        // Save to file
        // TODO: Choose path based on user input, or from options
        // TODO: Spawn a new thread to do this.
        emulator()->SaveToFile("test.sav");
      } break;
      case ui::VirtualKey::kF8: {
        // Restore from file
        // TODO: Choose path from user
        // TODO: Spawn a new thread to do this.
        emulator()->RestoreFromFile("test.sav");
      } break;
      #endif
```

Reply:
Yes, though possibly without indentation for `#if 0` and `#endif`, but clang-format will make it correct.

Reply:
Done.